### ISSUE TITLE
Quick Start

### ISSUE THREAD

**Issue by imfing** [imfing/issues-blog#3]
## Create your own repository 📁

Go to the repository, and click the **Use this template** button, then select **Create a new repository**.  Follow the instructions to create a new repository, feel free to make it either **Public** or **Private**.

<img width="400" src="https://docs.github.com/assets/cb-100333/images/help/repository/use-this-template-button.png">

For more information, please check out [Creating a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).

## Edit files 🔧

Once your repository is ready, modify the following places to your own `<user>/<repo>`:

1. [`go.mod`](https://github.com/imfing/issues-blog/blob/main/go.mod) file, modify the first line.
2. [`config/_default/config.toml`](https://github.com/imfing/issues-blog/blob/main/config/_default/config.toml), change the `baseUrl` accordingly.

## Create an issue ✍️

Start by adding a new issue to your repository. Use Markdown to format the content.

For further instruction, see [Creating an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) and [Basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

![image](https://user-images.githubusercontent.com/5097752/203459294-8062f640-ddd5-4d2d-8f8d-f2ec66e22229.png)

## Publish your site 🚀

Once it's finished, **Close** the issue. [GitHub Actions](https://docs.github.com/en/actions) should automatically start to build and publish your website. 

![GitHub Actions](https://docs.github.com/assets/cb-28129/images/help/images/learn-github-actions-repository.png)

**Enable [GitHub Pages](https://docs.github.com/en/pages)**

1. Go to **Settings**
  ![](https://docs.github.com/assets/cb-27528/images/help/repository/repo-actions-settings.png)
2. Go to **Pages** tab,  under "Build and deployment", under "Source", select **Deploy from a branch**. Choose `gh_pages` from the dropdown menu.
  ![image](https://user-images.githubusercontent.com/5097752/203459904-554f37ca-6451-468b-9801-df8ffd824014.png)

After finish the above steps, wait a while, your site should be live at `https://<your_name>.github.io/<your_repo>` 🎉
